arm/Microsoft.MachineLearningServices/workspaces/.parameters/parameters.json

```diff
@@ -17,6 +17,53 @@
         "associatedApplicationInsightsResourceId": {
             "value": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Insights/components/adp-<<namePrefix>>-az-appi-x-001"
         },
+        "systemAssignedIdentity": {
+            "value": true
+        },
+        "userAssignedIdentities": {
+            "value": {
+                "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-<<namePrefix>>-az-msi-x-001": {}
+            }
+        },
+        "computes": {
+            "value": [
+                {
+                    "name": "DefaultCPU",
+                    "location": "westeurope",
+                    "computeLocation": "westeurope",
+                    "sku": "Basic",
+                    "systemAssignedIdentity": true,
+                    "userAssignedIdentities": {
+                        "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-<<namePrefix>>-az-msi-x-001": {}
+                    },
+                    "description": "Default CPU Cluster",
+                    "disableLocalAuth": false,
+                    "computeType": "AmlCompute",
+                    "properties": {
+                        "enableNodePublicIp": true,
+                        "isolatedNetwork": false,
+                        "osType": "Linux",
+                        "remoteLoginPortPublicAccess": "Disabled",
+                        "scaleSettings": {
+                            "maxNodeCount": 3,
+                            "minNodeCount": 0,
+                            "nodeIdleTimeBeforeScaleDown": "PT5M"
+                        },
+                        "vmPriority": "Dedicated",
+                        "vmSize": "STANDARD_DS11_V2"
+                    }
+                }
+                // {
+                //     "name": "DefaultAKS",
```

Comment:
I guess this is commented because it cannot be deployed in parallel?

Reply:
@MrMCake The reason why it is commented is to show how it would work but it is not enabled as it would require an AKS cluster. Do we have an AKS as dependency available?

It is possible to deploy multiple compute resources in parallel.

Reply:
Not yet, no. But we can do this in the future with a new dependency approach. I'm usually not a big fan of these kind of comments - so could you please add a smal disclaimer on top if it saying : `This is an example of a link to an existing AKS resource`

Reply:
@MrMCake I removed the commented part.